bin/iqb

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/zsh
```

Comment:
Comes standard with OS X and a lot of linuxes so I'm not super concerned with it.